golang/cosmos/daemon/cmd/root.go

```diff
@@ -90,6 +92,7 @@ func appendToPreRunE(cmd *cobra.Command, fn cobraRunE) {
 func NewRootCmd(sender vm.Sender) (*cobra.Command, params.EncodingConfig) {
 	// we "pre"-instantiate the application for getting the injected/configured encoding configuration
 	// note, this is not necessary when using app wiring, as depinject can be directly used (see root_v2.go)
+	var emptyWasmOpts []wasmkeeper.Option
```

Comment:
Add:
```go
import (
  "github.com/CosmWasm/wasmd/x/wasm/keeper/wasmtesting"
)
```

And use `tempWasmOpts` in the arguments to `tempApp := gaia.NewSimApp` below.
```suggestion
	tempWasmEngine := &wasmtesting.MockEngine{}
	tempWasmOpts :=  []wasmkeeper.Option{wasmkeeper.WithWasmEngine(tempWasmEngine)}
```

That might crash, but it's a start.